doc/mgr/dashboard.rst

```diff
@@ -181,13 +181,16 @@ app.
 Username and password
 ^^^^^^^^^^^^^^^^^^^^^
 
-In order to be able to log in, you need to define a username and password, which
-will be stored in the MON's configuration database::
+In order to be able to log in, you need to create an user account and associate
+it with at least one role. We provide a set of predefined *system roles* that
+you can use. For more details please refer to the `User and Role Management`_
+section.
```

Comment:
Can you please also update the feature list on top of this document accordingly? I think it is worth highlighting that we now support multiple users and roles.